### ISSUE TITLE
ReferenceError: exports is not defined

### ISSUE THREAD

**Issue by chriswardo** [casesandberg/react-color#771]
es/helpers/color.js in the new update (2.19.1)

**Comment by matthias-ccri** [comment]
Same here. Workaround for me:
```
  "resolutions": {
    "react-color": "2.18.0"
  }
```

**Comment by Dan-Wood** [comment]
https://github.com/casesandberg/react-color/issues/759

**Comment by travsr** [comment]
Same error here. Like matthias I had to revert to v `2.18.0`
```
$ npm install react-color@2.18.0
```

**Comment by michaelbrewer** [comment]
Same here. Hopefully fixed soon?

**Comment by michaelbrewer** [comment]
> react-colour 2.18.1 still works for me

**Comment by bailinx** [comment]
Same here + 1

**Comment by qq05281201** [comment]
Same here + 1

**Comment by zalatmza** [comment]
Same here + 1，
hopefully fixed soon.

**Comment by chenshuai2144** [comment]
Same here + 1，

**Comment by jinboker** [comment]
Same here + 1

**Comment by mfmsajidh** [comment]
Same problem here

![image](https://user-images.githubusercontent.com/36571307/97389297-fb7e0800-18ff-11eb-90e9-910ae6c0fb55.png)

**Comment by solarstar101** [comment]
just got this issue also.... rever back to 2.18.0

**Comment by junqdu** [comment]
Same here + 1

**Comment by manuramber** [comment]
Same issue here. I had to paste an old version of react-colors on node modules folder. Im using 2.18.1

**Comment by avespoli-bf** [comment]
+1

**Comment by casesandberg** [comment]
This has been fixed and published as `react-color@2.19.3`

**Comment by shilman** [comment]
Thanks for the fast turnaround @casesandberg! You're awesome 💯
